packages/ember-routing/lib/services/router.js

```diff
@@ -0,0 +1,49 @@
+/**
+@module ember
+@submodule ember-routing
+*/
+
+import Service from 'ember-runtime/system/service';
```

Comment:
Imports should be from the entry points (e.g. `from 'ember-runtime';` instead of `from 'ember-runtime/somethign';`)